### PR TITLE
fixing regexp

### PR DESCRIPTION
**Pull request by prateekbh**
- Fixes the regexp for normal -> module js
IIUC the change is required due to the new closure compiler declares global?global:this in a different way

### CODE REVIEW

File: build-system/tasks/create-module-compatible-es5-bundle.js [hunk 1]
```diff
@@ -31,8 +31,7 @@ const gulp = $$.help(require('gulp'));
 exports.createModuleCompatibleES5Bundle = function(src) {
   return gulp.src('dist/' + src)
       .pipe($$.sourcemaps.init({loadMaps: true}))
-      .pipe($$.rename(src.replace(/\.js$/, '-module.js')))
-      .pipe($$.regexpSourcemaps(/(global\?global:\w*})\(this\)/, '$1(self)', 'module-global'))
+      .pipe($$.regexpSourcemaps(/(window.global\?window.global:\w*)this/, '$1self', 'module-global'))
```
Comment (erwinmombay) [inline build-system/tasks/create-module-compatible-es5-bundle.js]: just for my own knowledge how is the regexp name used (what is its usefulness)?

Comment (prateekbh) [reply build-system/tasks/create-module-compatible-es5-bundle.js]: @jridgewell , i wanted to ask that too

Comment (jridgewell) [reply build-system/tasks/create-module-compatible-es5-bundle.js]: I think it's the filename this new code will be associated with.

**Comment by mrjoro** [comment]
@googlebot are you stuck?

**Pull request merged**